pkg/controller/template/render_test.go

```diff
@@ -282,6 +275,11 @@ func TestGenerateMachineConfigs(t *testing.T) {
 			t.Fatalf("failed to generate machine configs: %v", err)
 		}
 
+		foundPullSecretMaster := false
+		foundPullSecretWorker := false
+		foundKubeletUnitMaster := false
+		foundKubeletUnitWorker := false
```

Comment:
this is the most stupid nits every but declaring a boolean in Golang is the same as setting it to false so `var found......, found .... , ..... bool` is the same here (and this nits can live here for the time being, just wanted to point it out :D